docs/tutorials/query-language.md

```diff
@@ -53,6 +53,58 @@ let peopleWhoseNameContainsA = realm.objects('Contact').filtered('name CONTAINS[
 let Johns = realm.objects('Contact').filtered('name ==[c] "john"');
 ```
 
+You can do simple wildcard matching with `LIKE` which supports using `?` to match a single character and `*` to match zero or multiple characters.
+
+Example:
+```JS
+// Matches "John" and "Johnny"
+let Johns = realm.objects('Contact').filtered('name LIKE "John*"');
+```
+
 ### Composition
 Use parentheses and the `&&`/`AND` and `||`/`OR` operators to compose queries. You can negate a predicate with `!`/`NOT`.
 
+### Queries on collections
+
+When objects contain lists you can query into them using the collection operators `ANY`, `ALL` and `NONE`.
```

Comment:
I discovered that `ALL` and `NONE` queries are matching objects with no items in the list which I believe is incorrect. A workaround is to append `AND friends.@count > 0` to `ALL` and `NONE` queries, but I wonder if we should fix this behaviour in core before documenting these?

Reply:
We should check what the NSPredicate behavior is here. cc @bdash